scripts/install_third_party.sh

```diff
@@ -38,9 +38,14 @@ install_node_module gulp-util 3.0.7
 install_node_module htmllint 0.7.2
 install_node_module htmllint-cli 0.0.7
 install_node_module @mapbox/stylelint-processor-arbitrary-tags 0.2.0
-install_node_module postcss-syntax 0.10.0
-install_node_module stylelint 9.2.1
+install_node_module postcss 5.0.0
+
```

Comment:
Please remove this empty line

Reply:
Hi @vojtechjelinek Thanks for the review! 
But right now I am desperately trying to get the tests pass. There seems to be some problem with the installation of third party node modules.
The extra line might have been left while debugging. Will remove that.
Thanks!